packages/app/src/components/TVChart/TVChart.tsx

```diff
@@ -300,6 +307,12 @@ export function TVChart({
 		})
 	}, [marketAsset])
 
+	useEffect(() => {
+		_widget.current?.subscribe('onAutoSaveNeeded', () => {
```

Comment:
Wondering if we need to store a ref to this and unsubscribe when we clear the widget?

Reply:
Literally made the change locally but hadn't pushed it. Pushed now.